.gitignore

```diff
@@ -33,6 +33,7 @@ _infrastructure/tests/build
 !scripts/new-package.js
 !scripts/not-needed.js
 !scripts/lint.js
+!scripts/update-codeowners.js
```

Comment:
Please revert this.

Reply:
I will revert now because this change is not related to rangy module.
But file `scripts/update-codeowners.js` is ignored according to the rules in `.gitignore` - I am sure this is a mistake.